pkg/db/dto/project.go

```diff
@@ -109,8 +112,26 @@ func ToGitStatusDTO(status *project.GitStatus) *GitStatusDTO {
 		return nil
 	}
 
+	var ahead, behind *int32
+	if status.Ahead != 0 {
+		value := int32(status.Ahead)
+		ahead = &value
+	}
+	if status.Behind != 0 {
+		value := int32(status.Behind)
+		behind = &value
+	}
+	var branchPublished *bool
+	if status.BranchPublished {
```

Comment:
Is this check necessary? Why don't we simply assign `BranchPublished: &status.BranchPublished`